### PR TITLE
Benchmark : version bump

### PR DESCRIPTION
**Pull request by DethroTull**
Previous recipe was for version 1.2.0, and according to https://repology.org/metapackages/?inrepo=haikuports_master&outdated=True, benchmark was outdated so I wanted to update in order to get my feet wet with the haikuports process.

### CODE REVIEW

**Comment by Begasus** [comment]
You should rename the topic to include the name of the project "benchmark" in this case ;)

File: dev-libs/benchmark/benchmark-1.3.0.recipe [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+INSTALL()
+{
+	cd build
+	make install
+
+	mkdir -p $includeDir
+	mv $prefix/include/* $includeDir/
+	rm -rf $prefix/include/
+
+	prepareInstalledDevelLibs libbenchmark
```
Comment (fbrosson) [inline dev-libs/benchmark/benchmark-1.3.0.recipe]:
```diff
-	prepareInstalledDevelLibs libbenchmark
+	prepareInstalledDevelLib libbenchmark
```
(prepareInstalledDevelLib**s** is required when we have more than one lib. 
Here, prepareInstalledDevel**Lib** is enough and will do the same work :)

File: dev-libs/benchmark/benchmark-1.3.0.recipe [hunk 2]
```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+HOMEPAGE="https://github.com/google/benchmark/"
```
Comment (fbrosson) [inline dev-libs/benchmark/benchmark-1.3.0.recipe]:
```diff
-HOMEPAGE="https://github.com/google/benchmark/"
+HOMEPAGE="https://github.com/google/benchmark"
```

File: dev-libs/benchmark/benchmark-1.3.0.recipe (hunk 1 above)
Comment (DethroTull) [reply dev-libs/benchmark/benchmark-1.3.0.recipe]: Very good to know!  Thank you for the explanation.

Comment (DethroTull) [inline dev-libs/benchmark/benchmark-1.3.0.recipe]: Ok, I've obviously done something incorrect here.  I'm going to remove this pull request and try again.

**Pull request closed without merging**